blocks/library/cover-image/style.scss

```diff
@@ -41,5 +42,44 @@
 	&.components-placeholder {
 		height: inherit;
 	}
+	
+	.button-container {
+		top: 100px;
+		font-family: $default-font;
+		display: inline-block;
+		text-decoration: none;
+		font-size: $big-font-size;
+		margin: 0;
+		padding: 0 24px;
+		height: $blocks-cover-image__button-height;
+		line-height: $blocks-cover-image__button-height;
+		cursor: default;
+		border-radius: $blocks-cover-image__button-height / 2;
+		white-space: nowrap;
+		background: $blue-medium-400;
+		color: $white;
+		vertical-align: top;
+		position: relative;
+	
+		a {
+			box-shadow: none !important;
+			color: $white;
+			cursor: pointer;
+			text-decoration: none !important;
+		}
+	
+		&:hover {
+			color: $white;
+		}
+	
+		&.aligncenter {
+			display: inline-block;
+		}
+	}
+
+}
 
+.wp-block-cover-image > h2 {
+	margin: 0px !important;
```

Comment:
Do we need `!important`? Is this an issue of [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)? We should avoid `!important` as much as possible.